pkg/minikube/node/cache.go

```diff
@@ -42,7 +42,12 @@ func beginCacheKubernetesImages(g *errgroup.Group, imageRepository string, k8sVe
 			glog.Info("Caching tarball of preloaded images")
 			return download.Preload(k8sVersion, cRuntime)
 		})
-		return
+		err := g.Wait()
```

Comment:
I think we can remove the previous call to g.Go(), and just turn it into an err check.